xml/operating_system_darwin.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Patterns for common names of Mac OS X operating systems.
+-->
+<fingerprints matches="operating_system.name">
+  <fingerprint pattern="^Mac OS X\s?(\d+?(?:\.\d+?)+?)?$" flags="REG_ICASE">
+    <description>Mac OS X with version number</description>
+    <example os.version="10.10.5">Mac OS X 10.10.5</example>
```

Comment:
Last I looked I think the examples are living test cases that get used to test each pattern. Is there a way to provide negative cases?